docs/html/user_guide.rst

```diff
@@ -49,6 +49,22 @@ For more information and examples, see the :ref:`pip install` reference.
 .. _PyPI: https://pypi.org/
 
 
+Basic Authentication Credentials
+********************************
+
+pip supports basic authentication credentials. Basically, in the url there is
+a username and password separated by ``:``.
+
+  ::
+
+    https://[username[:password]]@pypi.company.com/simple
+    0123456789abcdef@pypi.company.com
```

Comment:
I would preface this example with something like

> For indexes that only require single-part authentication tokens, provide the token as the "username" and do not provide a password, for example
>
>     https://acbd18db4cc2f85cedef654fccc4a4d8@pypi.company.com

Reply:
ok